### ISSUE TITLE
NVDA doesn't read french numbered values correctly

### ISSUE THREAD

**Issue by kimyouree** [nvaccess/nvda#10765]
### Steps to reproduce:
1. Open NVDA.

2. Set your language preference to English in the NVDA preferences menu. Open any text editor of your choosing and type a numbered value that has a suffix, such as: '75th' and '250th'. An example sentence could be: "Harry finished in 75th place and Sam finished in 250th place", then have the screen reader read that sentence.

3. Change your language preference to French in the NVDA preferences menu. And have the screen reader read that same sentence in french.

### Actual behavior:
In english, you will notice the numbered values are read with correct grammatical pronounciation, '75th' and '250th' will sound like: "seventy-fifth place" and "two-hundred-and-fiftieth place". It does not read them as "seven-five-thhh" and "two-five-oh-thhh".

However, NVDA does not read the same numbered values as nicely when the language is switched to French. There are two problems with numbers being read in French mode:

1. While in French mode, numbers are still read in english and if a numbered value has 2 or more digits, each digit is read one-by-one, instead of as a single value. "75" is read as "seven-five" and "250" is read as "two-five-oh".

2. Additionally, if the numbered value has a suffix, its not read correctly in combination to the numbers preceding it. Each character is read individually, instead of as one value, it sounds like: "seven-five-eee" and "two-five-oh-e".

### Expected behavior:

NVDA should read numbers correctly in a French context as it does in an English one to be equally reader-friendly in both languages. 

If the french equivalent of the values, "75th" and "250th", are written as: "75e" and "250e", they should be read to sound like: "soixante-quinzième" and "deux cent cinquantième", instead of "seven-five-eee" and "two-five-oh-eee".

#### NVDA installed/portable/running from source:
NVDA installed.

#### NVDA version:
2019.2.1

#### Windows version:
Windows 10 Pro. Version 1903. OS build: 18362.592
#### Name and version of other software in use when reproducing the issue:
Adobe Reader for reading pdfs but the same issue exists when reading numbered content in the browser. For example, I got this text from the website of the French Government, "La 3e édition du sommet Choose France, ce 20 janvier 2020 à Versailles, vise notamment à promouvoir l’attractivité de la France et à encourager les investissements étrangers au cœur de nos territoires."  This is from the link: [Governement of France Site](https://www.gouvernement.fr/choose-france-un-sommet-pour-conforter-l-attractivite-de-la-france)

**Comment by josephsl** [comment]
Hi, an important question: what is the speech synthesizer you are using? Partly this can be explained by which synthesizer is in use, as “sw3itching languages” in this context is actually changing NVDA’s interface language, not the speech itself. Thanks.

**Comment by kimyouree** [comment]
Hi Joseph, 

Thank you for your response, I have since changed my synthesizer to 'NGSpeak' (it was previously set to Windows OneCore Voices) and I have also set the voice to 'French (France)'. 
Doing so has definitely solved part 1 of the issue, which is wonderful, however, part 2 remains unresolved.

The issue that remains is: it reads '75e' as 'soixante-quinze-é' and '250e' as 'deux cent cinquante-é'.

Instead, it should read as: 'soixante-quinzième' and 'deux cent cinquantième'. Is there a another NVDA setting that you know of that reads it in this format? 

Thanks!

**Comment by josephsl** [comment]
Hi, as for the second one, I’m not sure, as I’m not a French speaker. Thanks.

**Comment by Adriani90** [comment]
I think for this you should raise an issue against eSpeak. They also have a repository on Github. There is nothing which can be done about this in NVDA itself.
I am closing this as invalid.

**Comment by kimyouree** [comment]
thanks for the tip! I will do that :) 


> I think for this you should raise an issue against eSpeak. They also have a repository on Github. There is nothing which can be done about this in NVDA itself.
> I am closing this as invalid.
